### PR TITLE
[Docs]Corrected error command in Quick Start Guide

### LINKED ISSUE #3049: [Docs]Error command in the Quick Start Guide
#### Current State

In the [Quick Start Guide](https://docs.meshery.io/installation/quick-start) under the Install Meshery section the command
`$ curl -L https://git.io/meshery | bash - `
gives an unexpected error as the platform (Docker, Kubernetes) is not defined.

#### Desired State

By running the single command in the Quick Start Guide under the Install Meshery section should download, install, and run Meshery.

---

#### Contributor Resources

- [Meshery documentation site](https://docs.meshery.io/)
- [Meshery documentation source](https://github.com/layer5io/meshery/tree/master/docs)
- [Instructions for contributing to documentation](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#documentation-contribution-flow)
- [Layer5 Community Slack](http://slack.layer5.io)

### PR DESCRIPTION
**Pull request by sayantan1413**
Signed-off-by: sayantan1413 <sayantanbose2001@gmail.com>

**Description**

Corrected the error command in the Quick Start Guide

This PR fixes #3049 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/meshery/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Review by leecalcote — APPROVED** [review]
Looks great. Thank you, @sayantan1413

**Comment by leecalcote** [comment]
@sayantan1413 will you update this description to "fixes #<issuenumber>"? This should be one line, not two lines otherwise GH will not auto-close the associated issue.
Screen Shot 2021-05-30 at 9 55 31 PM

**Comment by sayantan1413** [comment]
> @sayantan1413 will you update this description to "fixes #"? This should be one line, not two lines otherwise GH will not auto-close the associated issue.
> Screen Shot 2021-05-30 at 9 55 31 PM

@leecalcote I changed it can you verify once ?

File: docs/pages/installation/index.md [hunk 1]
```diff
@@ -16,12 +16,21 @@ Getting Meshery up and running locally on a Docker-enabled system or in Kubernet
...
+ <span style="font-size: 20px">`Install on Docker`</span>
  <pre class="codeblock-pre"><div class="codeblock">
  <div class="clipboardjs">
- $ curl -L https://git.io/meshery | bash - 
+ $ curl -L https://git.io/meshery | PLATFORM=docker bash - 
  </div></div>
  </pre>
+ <br>
 
+ <span style="font-size: 20px">`Install on Kubernetes`</span>
```
Comment (leecalcote) [inline docs/pages/installation/index.md:27]: @sayantan1413 ideally, we're avoiding inline style where possible. Did you try one of the smaller headings or bold here? How did they look?

**Review by leecalcote — CHANGES_REQUESTED** [review]

File: docs/pages/installation/index.md (hunk 1 above)
Comment (sayantan1413) [reply docs/pages/installation/index.md:27]: Okay no problem I am changing the **span** to **h6**  it looks almost similar.

File: docs/pages/installation/index.md [hunk 2]
```diff
@@ -16,12 +16,21 @@ Getting Meshery up and running locally on a Docker-enabled system or in Kubernet
 
 Use the Meshery command line interface, [mesheryctl]({{ site.baseurl }}/guides/mesheryctl), to install and start Meshery. Begin with Meshery by installing its command line client: [mesheryctl]({{ site.baseurl }}/guides/mesheryctl). If you are on a MacOS or Linux system, you can download, install, and run to the management plane with the command shown in the figure.
 
+ <span style="font-size: 20px">`Install on Docker`</span>
```
Comment (leecalcote) [inline docs/pages/installation/index.md:19]:
```suggestion
###### Install on Docker
```

File: docs/pages/installation/index.md (hunk 1 above)
Comment (leecalcote) [inline docs/pages/installation/index.md:27]:
```suggestion
 ###### Install on Kubernetes
```

Comment (leecalcote) [reply docs/pages/installation/index.md:27]: Thanks. The backticks are unnecessary, too. And should be reserved for code / cli only.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Pull request merged**